great_expectations/data_context/store/ge_cloud_store_backend.py

```diff
@@ -47,7 +47,42 @@ class ResponsePayload(TypedDict):
 AnyPayload = Union[ResponsePayload, ErrorPayload]
 
 
-def _get_user_friendly_error_message(
+def construct_url(
+    base_url: str,
+    organization_id: str,
+    resource_name: str,
+    id: Optional[str] = None,
+) -> str:
```

Comment:
Moved these out of the store backend but they could also be static methods? I'm okay with these sitting globally but lmk if you disagree

Reply:
Either static or module level works. These module level functions work for me :)

Reply:
Do we really need this helper function at all. It seems just to obfuscate the URL, without giving us anything in return.

Reply:
this is the logic used in `GeCloudStoreBackend` to construct URLs - this function simply wraps it so we can reuse more easily